com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Debug-Window.md

```diff
@@ -117,28 +136,28 @@ The **Lighting** panel has tools that you can use to visualize various component
 | **- Punctual Lights**                 | Enable the checkbox to see [Punctual Lights](Glossary.md#PunctualLight) in your Scene. Disable this checkbox to remove Punctual Lights from your Scene's lighting. |
 | **- Area Lights**                     | Enable the checkbox to see Area Lights in your Scene. Disable this checkbox to remove Aera Lights from your Scene's lighting. |
 | **- Reflection Probes**               | Enable the checkbox to see Reflection Probes in your Scene. Disable this checkbox to remove Reflection Probes from your Scene's lighting. |
+| **Exposure**                          | Allows you to select an [Exposure](Override-Exposure.md) debug mode to use. |
+| **- Debug Mode**                      | Use the drop-down to select a debug mode. See [Exposure](Override-Exposure.md#exposure-debug-modes) documentation for more information. |
+| - **Show Tonemap curve**              | Enable the checkbox to overlay the tonemap curve to the histogram debug view.<br/>This property only appears when you select **HistogramView** from **Debug Mode**. |
+| **- Center Around Exposure**          | Enable the checkbox to overlay the center the histogram around the current exposure value.<br/>This property only appears when you select **HistogramView** from **Debug Mode**. |
+| **- Display RGB Histogram**           | Enable the checkbox to display the Final Image Histogram as an RGB histogram instead of just luminance.<br />This property only appears when you select **FinalImageHistogramView** from **Debug Mode**. |
+| **- Display Mask Only**               | Enable the checkbox to display only the metering mask in the picture-in-picture, if unchecked the mask will displayed after weighting the scene color instead. <br />This property only appears when you select **MeteringWeighted** from **Debug Mode**. |
```

Comment:
Enable the checkbox to display only the metering mask in the picture-in-picture. When disabled, the mask displays after weighting the scene color instead. <br />This property only appears when you select **MeteringWeighted** from **Debug Mode**.